client/src/views/LoginPage.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="h-screen w-screen flex-col bg-gradient-to-br from-cyan-400 to-green-500 pt-10 font-header">
+    <div class="ml-auto mr-auto h-1/6 w-1/2 items-center justify-center rounded-3xl bg-yellow-50">
+      <p class="pt-8 text-4xl font-semibold leading-10">Welcome to your To-do App</p>
+    </div>
+    <div class="m-auto mt-10 h-1/3 justify-center rounded-3xl bg-yellow-50 phone:w-8/12 computer:w-4/12">
+      <p class="ml-auto mr-auto w-full pt-4 text-3xl font-semibold">Login</p>
+
+      <UserInput :label="'Email:'" :property="'email'" v-model="email" />
```

Comment:
You finished the whole front end part of the Todo project, now you have to create a Login page and we still see stuff like this... Please pay more attention.